File: src/main/java/utils/TripDTO.java
package utils;

import java.sql.Time;

public class TripDTO {
    private int id;
    private final String touristAttraction;
    private final String transportCompany;
    private final Time departureTime;
    private final double price;
    private final int availableSeats;

    public TripDTO(int id, String touristAttraction, String transportCompany, Time departureTime, double price, int availableSeats) {
        this.id = id;
        this.touristAttraction = touristAttraction;
        this.transportCompany = transportCompany;
        this.departureTime = departureTime;
        this.price = price;
        this.availableSeats = availableSeats;
    }

    public TripDTO(String touristAttraction, String transportCompany, Time departureTime, double price, int availableSeats) {
        this.touristAttraction = touristAttraction;
        this.transportCompany = transportCompany;
        this.departureTime = departureTime;
        this.price = price;
        this.availableSeats = availableSeats;
    }

    public String getTouristAttraction() {
        return touristAttraction;
    }

    public String getTransportCompany() {
        return transportCompany;
    }

    public Time getDepartureTime() {
        return departureTime;
    }

    public double getPrice() {
        return price;
    }

    public int getAvailableSeats() {
        return availableSeats;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }
}

File: src/utils/TripDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace AgentiiDeTurism.src.utils
{
    public class TripDTO
    {
        public int Id { get; set; }
        public string TouristAttraction { get; set; }
        public string TransportCompany { get; set; }
        public TimeSpan DepartureTime { get; set; }
        public double Price { get; set; }
        public int AvailableSeats { get; set; }

        public TripDTO(int id, String touristAttraction, String transportCompany, TimeSpan departureTime, double price, int availableSeats)
        {
            this.Id = id;
            this.TouristAttraction = touristAttraction;
            this.TransportCompany = transportCompany;
            this.DepartureTime = departureTime;
            this.Price = price;
            this.AvailableSeats = availableSeats;
        }

        public TripDTO(String touristAttraction, String transportCompany, TimeSpan departureTime, double price, int availableSeats)
        {
            this.TouristAttraction = touristAttraction;
            this.TransportCompany = transportCompany;
            this.DepartureTime = departureTime;
            this.Price = price;
            this.AvailableSeats = availableSeats;
        }
    }
}
